Sources/Main.cpp

```diff
@@ -177,6 +177,100 @@ void TX584Form::LoadPRJ(TFileStream *Stream)
         CodeListView->Items->Item[i]->SubItems->Strings[3] = L"";
     }
 }
+
+void TX584Form::LoadX584(TFileStream *Stream)
```

Comment:
This branch now contains two copies of `LoadX584` and `LoadPRJ`. I guess something went wrong during the merge.